src/wasm.h

```diff
@@ -1383,6 +1383,8 @@ class Table : public Importable {
 class Memory : public Importable {
 public:
   static const Address::address_t kPageSize = 64 * 1024;
+  static_assert(IsPowerOf2(kPageSize), "kPageSize should be power of two");
```

Comment:
I don't think this is helpful - this is a fixed part of the wasm spec, and will never change, really. So if an assert were needed, it could be "is 64K" specifically. But I think that's excessive, as we test it in other ways.